src/tlo/methods/rti.py

```diff
@@ -4004,6 +4028,11 @@ def apply(self, person_id, squeeze_factor):
             logger.debug(key='rti_general_message',
                          data=f"Fracture casts available for person {person_id} {open_fracture_counts} open fractures"
                          )
+
+            # TODO: open fractures are usually treated surgically so more equipment is likely required.
+            #  Not immediately clear if these individuals are also scheduled to the surgical HSIs. will leave for now
```

Comment:
Is this smt we could ask @andrew-phillips-1, or will you look at this?

Reply:
I will just add surgical equipment for now as thats most appropriate.